apiserver/facades/controller/caasunitprovisioner/provisioner.go

```diff
@@ -218,15 +217,10 @@ func (a *Facade) UpdateApplicationsUnits(args params.UpdateApplicationUnitArgs)
 			continue
 		}
 
-		// Get the app config so we know how to handle the update.
-		// It depends on whether Juju or the cloud manage the units.
-		appConfig, err := app.ApplicationConfig()
-		if err != nil {
-			result.Results[i].Error = common.ServerError(err)
-			continue
-		}
-
-		if appConfig.GetBool(caas.JujuManagedUnits, caas.JujuDefaultJujuManagedUnits) {
+		// always set to `false` to disable `jujuManagedUnits`
+		jujuManagedUnits := false
+		if jujuManagedUnits {
+			// TODO: remove here?
 			err = a.updateJujuManagedUnitsFromCloud(app, appUpdate.Units)
```

Comment:
Yes, since there is now only one code path, I do not see the value of having an if-statement....